__tests__pet.test.js

```diff
@@ -0,0 +1,99 @@
+const Pet = require('./src/pet');
+
+describe ('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object)
+    })
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido')
+
+        expect(pet.name).toEqual('Fido')
+    })
+
+    it('has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.age).toEqual(0);
+    })
+
+    it('has an initial hunger of 0', () => {
+        const pet = new Pet('Dash');
+
+        expect(pet.hunger).toEqual(0);
+    })
+
+    it('has an initial fitness of 10', () => {
+        const pet = new Pet('Flimsy');
+
+        expect(pet.fitness).toEqual(10);
+    })
+
+    it('changes properties correctly when growUp() is called', () => {
+        const pet = new Pet('Doggie');
+        pet.growUp();
+
+        expect(pet.age).toEqual(1);
+        expect(pet.hunger).toEqual(5);
+        expect(pet.fitness).toEqual(7);
+    })
+
+    it('increases fitness by 4, but no more than 10, when walk() is called', () => {
+        const pet = new Pet('Peppa');
+        pet.growUp(); // fitness is now 7, hunger is 5
```

Comment:
Slight issue with the comments here, in that if the values for things like max hunger or the increment levels change, all your comments will become out of date. Generally its a good idea to use variables to store these values so you can reference them instead